.nvmrc

```diff
@@ -0,0 +1 @@
+v16.19.0
```

Comment:
hmmmm
I remember having issues with some projects with this node version. Why are we moving to this one for this project ?

Reply:
My bad, replied on the wrong comment: https://github.com/upfluence/oss-components/pull/350#discussion_r1461815512.